src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -24,6 +24,7 @@ export class FeederPodcastModel extends BaseModel {
   hasPublicFeed: boolean = false;
 
   VALIDATORS = {
+    category: [REQUIRED()],
     link: [REQUIRED(), URL('Not a valid URL')],
```

Comment:
let's add `explicit: [REQUIRED()],` here too, empty string/option should not be considered a valid value